sdk/storage/azure_storage_blob/src/lib.rs

```diff
@@ -15,7 +15,7 @@ pub use clients::*;
 
 pub mod models {
     pub use crate::generated::models::{
-        AccessTier, ArchiveStatus, BlobClientDeleteOptions, BlobClientDownloadOptions,
+        AccessTierOptional, ArchiveStatus, BlobClientDeleteOptions, BlobClientDownloadOptions,
```

Comment:
Nit: why the prefix "optional"? Nothing about its usage says "optional". Seems a bit confusing.